### ISSUE TITLE
Compile sources

### ISSUE THREAD

**Issue by jswelker** [jswelker/library-web-guidelines#4]
The first stage of the project is researching best practices and standards. We need to compile sources related to topics we will be discussing. Here are examples of acceptable sources:
- Published standards (e.g. WCAG, UAAG)
- Books
- Journal articles
- Studies published on the web

Add a citation to your sources on the appropriate page on the [wiki](https://github.com/jswelker/library-web-guidelines/wiki) using any citation format you'd like (I've been using APA to start). Be specific with your citations, especially if it's in a book. Narrow it down to a specific chapter or section if you can, or add an annotation after the citation. If the wiki is missing a heading, or if you don't like how the headings are organized, feel free to change them.

We're all (mostly) librarians, so I will assume we all know how to determine the quality of sources. The only thing I will add on that subject is that many of the [guidelines listed at usability.gov](http://guidelines.usability.gov/) cite _very_ dated information that might no longer be applicable (pre-mobile, pre-web 2.0, pre-Google, some even pre-Internet). Exercise judgment before including those.

Let's try to have this done by the end of January.

**Comment by jswelker** [comment]
Thanks to those who have started fleshing this out. Only about 4 people of the 25 of us have contributed to the research part of this project at this point. I blame eggnog and holiday shopping. I'm @including everyone on this issue as a reminder in case you are like me and had your brain go all fuzzy in the last several weeks.

@michaelschofield @vsiler @LisaGayhart @csheldonhess @hklish01 @cboman @jtidal @respinel @sciencex @knieper @rachel1011 @gnosisgithub @webgoddess @ahaines @stephenfrancoeur @holwhite @jaimemccurry @asteino @zoechao @emilykingatcsn @lkwiegand @deirdrelyon @zchartman @kooje

**Comment by michaelschofield** [comment]
Thanks for the tickler, @jswelker! I admit I totally forgot ---_---. This really isn't a shady plug, but there are some posts on [LibUX](//www.libux.co) that address topics we're concerned with here, and I've linked-up a ton of resources. When it comes to a best practice like "Links Should Open in the Same Window" (recently brought up on ALA Think Tank, so it's something librarians think about), we can rip-off the bibliography.
